samples/snippets/src/main/java/pubsublite/SubscriberExample.java

```diff
@@ -110,16 +106,14 @@ public void receiveMessage(PubsubMessage message, AckReplyConsumer consumer) {
     System.out.println("Listening to messages on " + subscriptionPath.value() + " ...");
 
     try {
-      System.out.println(subscriber.state());
       // Wait 30 seconds for the subscriber to reach TERMINATED state. If it encounters
-      // unrecoverable errors before then, its state will change to FAILED and
-      // an IllegalStateException will be thrown.
+      // unrecoverable errors before then, its state will change to FAILED and an
+      // IllegalStateException will be thrown.
       subscriber.awaitTerminated(30, TimeUnit.SECONDS);
     } catch (TimeoutException t) {
-      // Shut down the subscriber. This will change the state of the
-      // subscriber to TERMINATED.
+      // Shut down the subscriber. This will change the state of the subscriber to TERMINATED.
       subscriber.stopAsync();
```

Comment:
you should still probably call AwaitTerminated after calling stopAsync

Reply:
Done.